### PR TITLE
Docker-compose #339

### PR DESCRIPTION
**Pull request by fbalicchia**
Hey, first of all thanks for interesting project and I'm newbie with PyGrid and trying to build 
docker image and use docker-compose but I'm experiencing in some errors.
Align env variable to actual code base docker-compose works for me
Please review my PR.
Thanks

### CODE REVIEW

**Comment by IonesioJunior** [comment]
Hello @fbalicchia, this is amazing!

File: docker-compose.yml [hunk 1]
```diff
@@ -3,39 +3,40 @@ services:
...
         image: node:latest
         environment:
-                - GRID_NETWORK_URL=http://localhost:8080
+                - GRID_NETWORK_URL=http://localhost:5000
                 - ID=Bob
                 - ADDRESS=http://localhost:3000/
-                - TEST_CONFIG=sqlite:///databaseBob.db
-                - PORT=3000
+                - DATABASE_URL=sqlite:///databaseBob.db
+                - GRID_WS_PORT=3000
```
Comment (IonesioJunior) [inline docker-compose.yml]: If we keep the PORT variable pattern we will have greater compatibility with different systems/platforms  (Heroku use as default $PORT to setup ports).

File: gateway/requirements.txt [hunk 2]
```diff
@@ -14,4 +14,4 @@ requests
 flask_sqlalchemy
 flask_migrate
 flask_testing
-psycopg2
```
Comment (IonesioJunior) [inline gateway/requirements.txt]: Could you add `psycopg2-binary` package?

Comment (fbalicchia) [reply gateway/requirements.txt]: done

File: docker-compose.yml (hunk 1 above)
Comment (fbalicchia) [reply docker-compose.yml]: done

**Pull request merged**